invenio_db/utils.py

```diff
@@ -28,7 +28,7 @@
 from flask import current_app
 from sqlalchemy.engine import reflection
 
-from invenio_db import db
+import invenio_db
```

Comment:
This had to be changed because it was causing a circural import with the `from .utils import create_alembic_verson_table` in cli.py